ddmrp/views/purchase_order_line_view.xml

```diff
@@ -19,14 +20,15 @@
                     name="on_hand_percent"
                     options='{"buffer_ids": "buffer_ids", "color_from": "execution_priority_level"}'
                 />
+                <field name="product_qty" readonly="True" />
                 <field name="product_id" readonly="True" />
                 <field
                     name="buffer_ids"
                     string="Buffers"
                     widget="many2many_tags"
                     readonly="1"
                 />
-                <field name="date_planned" widget="date" readonly="True" />
+                <field name="date_planned" widget="date" />
```

Comment:
Is it timezone safe having widget="date" when non readonly? On the PO editable tree view, there is no widget="date"

Reply:
@jbaudoux I don't know what you mean ? Could you explain ?

Reply:
I think he means that since the field is a `Datetime`, the fact of using the widget `date` could lead to unexpected results. 

I have tested it and indeed weird thing can happen: if you select say "2021-05-25" you are writing "2021-05-25 00:00 UTC" in the database, and if you are in a "negative" timezone like the US, you will see that after saving the date, it is displayed as "2021-05-24" (conversion to your timezone of the previous UTC datetime).

See below a gif of my testing:
![Peek 2021-05-05 11-09](https://user-images.githubusercontent.com/23449160/117119635-a3cec680-ad92-11eb-8671-2f7c5a896649.gif)

Nice catch @jbaudoux !!

What should we do? Just removing the widget might be the easiest...